source/portable/os/ota_os_freertos.c

```diff
@@ -155,21 +160,27 @@ OtaErr_t OtaReceiveEvent_FreeRTOS( OtaEventContext_t * pContext,
     return otaErrRet;
 }
 
-void OtaDeinitEvent_FreeRTOS( OtaEventContext_t * pContext )
+OtaErr_t OtaDeinitEvent_FreeRTOS( OtaEventContext_t * pContext )
 {
     ( void ) pContext;
 
+    OtaErr_t otaErrRet = OTA_ERR_NONE;
```

Comment:
I remember seeing build warning/failure on some platform if the first code statement is before all the local declarations. Consider moving the assert after all the local variables are declared i.e.
```c
OtaErr_t otaErrRet = OTA_ERR_NONE;

( void ) pContext;
```

Reply:
Addressed in https://github.com/aws/ota-for-aws-iot-embedded-sdk/pull/91/commits/5b69254cbead6a8843cb66190090e39e9940e6ce